build_runner/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.10.2
+
+- `--live-reload` cli option is replaced with `--hot-reload` one with appropriate
```

Comment:
I'm not sure if it will be very useful, because hot-reload already falls back to live-reload in complicated situations. If we still need to maintain both, we need to find good way to pass this option to client-side code

Reply:
can you elaborate on how and when the fallback happens? Do we have that documented somewhere?

Reply:
There are three cases when page reload might happen:
1. Dependency graph is changed since original page load. Documented in `Known issues`
2. Bubbling up from parent. We have default implementation of `hot$onChildUpdate` hook, so in current implementation it can't really happen. Documented only in test cases.
3. Either `hot$onChildUpdate`or `hot$onSelfUpdate` returns `false`. Not really a fallback but rather feature. Documented in hooks description.

Reply:
Ok. I will return live-reload flag in next PR